apps/platform/src/providers/MessageTriggerService.ts

```diff
@@ -125,3 +153,18 @@ export const throttleSend = async (channel: Channel, points = 1): Promise<RateLi
         },
     )
 }
+
+export const notifyJourney = async (user_step_id: number, response?: any) => {
+
+    // save response into user step
+    if (response) {
+        await JourneyUserStep.update(q => q.where('id', user_step_id), {
+            data: {
+                response,
+            },
+        })
+    }
```

Comment:
So all data for an entire iteration is being stored on the entrance step? Why not store on each individual step?

Reply:
this is the action step actually